File: commonlib/src/main/java/com/sixiangtianxia/commonlib/network/Logger.java
package com.sixiangtianxia.commonlib.network;

import com.sixiangtianxia.commonlib.utils.LogUtil;

public class Logger implements LoggingInterceptor.Logger {

    @Override
    public void log(String message) {
        LogUtil.Companion.i("54321",message);
    }

    @Override
    public void logI(String message) {
        LogUtil.Companion.i("54321",message);
    }
}

File: commonlib/src/main/java/com/sixiangtianxia/commonlib/network/Logger.kt
package com.sixiangtianxia.commonlib.network

import com.sixiangtianxia.commonlib.utils.LogUtil

class Logger : LoggingInterceptor.Logger {

    override fun log(message: String) {
        LogUtil.i("54321", message)
    }

    override fun logI(message: String) {
        LogUtil.i("54321", message)
    }
}
